src/main/java/be/seeseemelk/mockbukkit/inventory/meta/BundleMetaMock.java

```diff
@@ -0,0 +1,96 @@
+package be.seeseemelk.mockbukkit.inventory.meta;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.BundleMeta;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BundleMetaMock extends ItemMetaMock implements BundleMeta
+{
+
+	private List<ItemStack> items = null;
```

Comment:
Wouldn't it be simpler to just initialise this by-default to an empty ArrayList? That way we wouldn't have to write all those null-checks